app/models/custom_attribute.rb

```diff
@@ -1,4 +1,5 @@
 class CustomAttribute < ApplicationRecord
+  ALLOWED_API_FIELD_TYPES = %w(DateTime Time Date).freeze
```

Comment:
Since this requires some explanation, could you add a comment here to help make that clear?

Should this be then, `ALLOWED_API_VALUE_TYPES`?